dev/AnimatedVisualPlayer/AnimatedVisualPlayer.h

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+#include "common.h"
```

Comment:
pch.h and common.h should be included in the cpp file, not the headers.

Reply:
Not really. Might these includes be in pch in more recent master commit?

Reply:
I think you misunderstood my comment. The AnimatedVisualPlayer.h file should not have the lines #include "pch.h" or #include "common.h" -- those belong in the .cpp file.